github/apps_installation_test.go

```diff
@@ -11,6 +11,8 @@ import (
 	"net/http"
 	"reflect"
 	"testing"
+
+	"github.com/google/go-github/github"
```

Comment:
This is an internal package test .go file. Therefore, you don't need to import that package - you're already in it.

Instead of writing `github.ErrorResponse` below, you can access it with just `ErrorResponse`.

That will fix the problem.

You can read more about internal/external tests at https://golang.org/cmd/go/#hdr-Test_packages.